openfisca_uk/variables/finance/benefit/disability.py

```diff
@@ -36,118 +38,102 @@ class is_severely_disabled_for_benefits(Variable):
     value_type = bool
     entity = Person
     label = u"Whether this person is severely disabled for benefits purposes"
-    definition_period = WEEK
+    definition_period = YEAR
     reference = "Child Tax Credit Regulations 2002 s. 8"
 
     def formula(person, period, parameters):
         benefit = parameters(period).benefit
-        threshold_safety_gap = 10
+        THRESHOLD_SAFETY_GAP = 10 * WEEKS_IN_YEAR
```

Comment:
This should really be documented, but from memory it's because there are some small variations around the disability benefits paid, probably because of weeklyisation/irregular payments. But where we're imputing the basic/higher levels of benefits based on their amounts, we err on the side of being generous (£10/week is less than the difference between the levels).